source/slang/slang-diagnostic-defs.h

```diff
@@ -740,6 +740,8 @@ DIAGNOSTIC(41021, Error, differentiableFuncMustHaveOutput, "a differentiable fun
 DIAGNOSTIC(41022, Error, differentiableFuncMustHaveInput, "a differentiable function must have at least one differentiable input.")
 DIAGNOSTIC(41023, Error, getStringHashMustBeOnStringLiteral, "getStringHash can only be called when argument is statically resolvable to a string literal")
 
+DIAGNOSTIC(41030, Warning, operatorShiftLeftOverflow, "left shift amount, `$1`, is bigger than the bit count of type, `$0`.")
```

Comment:
how about?
"left shift amount, `$1`, exceeds the maximum number of bits supported by the data type, `$0`.")

Reply:
I like "exceeds" and "number of" is probably more natural in English expression than "count of".
I will change the message.
But there is no "maximum" nor minimum for the number of bits, so I will skip that word.

I am little more concerned about showing the "$1" in the middle of the message tho.
Probably I should move it to the end of the line like this,
```
left shift amount exceeds the number of bits, (`$0` << `$1`).
```